rllib/algorithms/algorithm_config.py

```diff
@@ -925,7 +925,7 @@ def validate(self) -> None:
         if self.simple_optimizer is True:
             pass
         # Multi-GPU setting: Must use MultiGPUTrainOneStep.
-        elif self.num_gpus > 1:
+        elif not self._enable_learner_api and self.num_gpus > 1:
```

Comment:
If `self._enable_learner_api `, we normally ignore `num_gpus`.
This line also makes this true for `num_gpus>1`

Reply:
(This does not fix any test, only the inaccuracy in https://github.com/ray-project/ray/pull/36045/files#r1214884043 lead to this throwing an error)